api/models/days_time_slots.py

```diff
@@ -17,7 +22,24 @@ class TimeSlot(models.Model):
     from_utc_hour = models.TimeField()
     to_utc_hour = models.TimeField()
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["from_utc_hour", "to_utc_hour"], name="from_to_hour")
+        ]
+
+    def __str__(self):
+        return f"{self.from_utc_hour.strftime('%H:%M')}-{self.to_utc_hour.strftime('%H:%M')} UTC"
```

Comment:
I don't know that yet, but it's possible. I guess it will a common thing not only in models, but also in views. Maybe not a constant, but a helper function that applies `.strftime("%H:%M")` to a `datetime` object?

Reply:
It's definitely good idea to have some helper class for working with dates - formatting, getting today date, etc. I think that's not very necessary now, but keep it in mind for future.

Reply:
👍🏻